model/models.go

```diff
@@ -0,0 +1,60 @@
+package model
+
+import (
+	"fmt"
+	"strings"
+)
+
+const instanceLabelFmt = "_%s_Instance"
+
+// DimensionsExtractedEvent represents a 'Dimensions Extracted' kafka messagae.
+type DimensionsExtractedEvent struct {
+	FileURL    string `avro:"file_url"`
+	InstanceID string `avro:"instance_id"`
+}
+
+// Dimension struct encapsulating Dimension details.
+type Dimension struct {
+	DimensionID string `json:"dimension_id"`
+	Value       string `json:"value"`
+	NodeID      string `json:"node_id,omitempty"`
+}
+
+// GetLabel returns the a formatted label for the dimensions in the format '_$DIMENSION_ID'
```

Comment:
Don't really need getters and setters, just store the correctly formatted var as part of the struct. Don't put json tags on them if we dont need to serialize them.